data-processor/src/main/java/io/micronaut/data/processor/mappers/jpa/jx/TableAnnotationMapper.java

```diff
@@ -54,14 +54,16 @@ public List<AnnotationValue<?>> map(AnnotationValue<Annotation> annotation, Visi
         annotation.stringValue(SqlMembers.CATALOG).ifPresent(catalog -> builder.member(SqlMembers.CATALOG, catalog));
         annotation.stringValue("schema").ifPresent(schema -> builder.member("schema", schema));
         final AnnotationValueBuilder<Indexes> idxBuilder = AnnotationValue.builder(Indexes.class);
-        Optional.ofNullable((AnnotationValue<Annotation>[]) annotation.getValues().get("indexes"))
-                .ifPresent(indexes -> {
-                    final AnnotationValue<Index>[] annotationValues =
-                            (AnnotationValue<Index>[]) Arrays.stream(indexes)
-                                    .map(a -> mapper.map(a, null)
-                                    .get(0)).toArray(AnnotationValue[]::new);
-                   idxBuilder.member("value", annotationValues);
-                });
+
+        Object indexesValue = annotation.getValues().get("indexes");
```

Comment:
I wonder if `annotationValues` can be used here

Reply:
@dstepanov fixed